StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs

```diff
@@ -67,7 +66,7 @@ public class SA1121UnitTests : CodeFixVerifier
                 }
                 catch (Exception ex)
                 {
-                    throw new AssertFailedException("Type failed: " + item.Item1 + Environment.NewLine + ex.Message, ex);
+                    Assert.True(false, "Type failed: " + item.Item1 + Environment.NewLine + ex.Message);
```

Comment:
:question: Any idea why this `try`/`catch` block is here? (Same as my previous question)